README.md

```diff
@@ -1,4 +1,5 @@
 # OpenSeadragon
+# ![Mascot](https://user-images.githubusercontent.com/5439169/65790675-0242b600-e115-11e9-817f-e31c41bf2ece.png)
```

Comment:
Image should go before the repo name so they end up on the same lie: `# ![Alt text](Mascot Image) Repository Name`